mysql_con.js

```diff
@@ -2,6 +2,9 @@ var mysql           = require("mysql");
 var express         = require("express");
 var app             = express();
 var bodyParser      = require("body-parser");
+var crypto          = require("crypto");
+var gets            = require("./app/mysql/get.js");
+var posts           = require("./app/mysql/post.js");
 
```

Comment:
I added the gets file before I realized I couldn't search the database for occurrences for a get; the 'data' that I passed didn't get registered. Which is why they're all posts. I'll need the get functionality later, so I left the connection there.